chrome/content/zotero/zoteroPane.js

```diff
@@ -4563,6 +4573,13 @@ var ZoteroPane = new function()
 	}
 	
 	
+	this.displayMissingImagesMessage = function () {
```

Comment:
`displayMissingNoteImageMessage` (+ "Note" and singular "Image", since this appears if there's a single missing image)